science/tetgen/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+
+name                    tetgen
+version                 1.6.0
+revision                1
+categories              science
+license                 AGPL-3
+maintainers             {xyloid.org:dave @mdave} openmaintainer
+description             A Quality Tetrahedral Mesh Generator and a 3D Delaunay Triangulator
+long_description        TetGen is a program to generate tetrahedral meshes of any 3D polyhedral \
+                        domains. TetGen generates exact constrained Delaunay \
+                        tetrahedralizations, boundary conforming Delaunay meshes, \
+                        and Voronoi partitions.
+
+platforms               darwin
+
+homepage                http://wias-berlin.de/software/tetgen/
```

Comment:
All links should be `https://` where possible.

Reply:
I updated the homepage which works fine, but unfortunately source appears to be available via `http` only.